src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarWriter.Unix.cs

```diff
@@ -41,13 +41,13 @@ public sealed partial class TarWriter : IDisposable
                 _ => throw new FormatException(string.Format(SR.TarInvalidFormat, Format)),
             };
 
-            if ((entryType is TarEntryType.BlockDevice or TarEntryType.CharacterDevice) && status.Dev > 0)
+            if ((entryType is TarEntryType.BlockDevice or TarEntryType.CharacterDevice) && status.RDev > 0)
```

Comment:
Why the `RDev > 0` check?

Reply:
I guess they aren't needed if the docs don't say they could have an unexpected value. I'll remove them.